frozendict/__init__.py

```diff
@@ -18,31 +18,35 @@ class frozendict(collections.Mapping):
     An immutable wrapper around dictionaries that implements the complete :py:class:`collections.Mapping`
     interface. It can be used as a drop-in replacement for dictionaries where immutability is desired.
     """
-
+    
     dict_cls = dict
-
+    
+    @classmethod
+    def fromkeys(cls, seq, value=None, *args, **kwargs):
+        return cls(dict.fromkeys(seq, value, *args, **kwargs))
```

Comment:
Does this need to be `return cls(cls.dict_cls.fromkeys(seq, value, *args, **kwargs))` so that `FrozenOrderedDict.fromkeys` still works as expected? :man_shrugging:
(I dunno, I haven't tested)

Reply:
I do not understand what you mean. Furthermore, FrozenOrderedDict is no more needed, since `dict` is ordered by default now, from Python 3.6 if I remember well.

Anyway, I created [my version of frozendict](https://github.com/Marco-Sulla/python-frozendict), that is more immutable, has more features and it's more fast. Lezica accepted to leave me the Pypi project, but I had no response from a while.

Furthermore, in the meanwhile, I decided to implement it in CPython and propose to add it to builtins of Python itself. The implementation is almost finished, but I currently have no time.

Reply:
> Lezica accepted to leave me the Pypi project, but I had no response from a while.

Oooh, you may be interested in the recent discussion in #25 then :slightly_smiling_face: